### DISCUSSION TITLE
What else is better than solving a problem

### DISCUSSION THREAD

**Discussion by snipex564** [community/community#137838] [New to GitHub]
### Select Topic Area

Question

### Body

I want to learn coding,I want to find a problem and solve it this is just the start we all have that wish but is it a dream or a desire then ur in the right place GitHub, GitHub isn't only about sharing codes launching web socket and other stuff it's all about finding and solving problems together yes I might just be talking to my self and maybe someone out there will ignore what I'm writing but it doesn't matter as long as u ask questions as people to help you learn from them share with them look for problem solve them and that is the beauty of GitHub Community

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @snipex564 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Feed category is solely related to conversations around the GitHub product `Feed`. This question should be in the `New to GitHub` category. I've gone ahead and moved it for you. Good luck!

**Comment by pranavbalu1** [comment]
You're not talking to urself, I'm here! I think when github was made they probably didnt think to build a full fledged community like this, but here we are.

**Reply by pranavbalu1** [reply]
can u mark me as answer ;)

**Comment by jovanovic-djo** [comment]
Short answer: Be consistent and have fun.

I would suggest you to start with small projects, commit daily to Github, explore, in order to find the tech stack you are interested in.

Also, doing interview problem questions (Data Structures and Algorithms) could be good warm up for an individual in the beginning. 

But still, I would prioritize working on personal projects and commit to other open source projects that I found interesting, and as a third option, do LeetCode or HackerRank.
